### ISSUE TITLE
how to connect movidius with win7

### ISSUE THREAD

**Issue by shuxp** [donnaware/movidius_windows#3]
I have installed NCSDK and also compiled my own graph.
So how can the movidius communicate with my computer? 
1， My computer is win7, I can't figure out how does movidius connect to win7 using code, could please share more about this?  ( In fact, BCB6 code is a bit difficult for me......)
2,    In my opinion, it can be done by open the device, load the graph, and then predict, but no samples, I don't know where to start.

thanks a ton for your help.

**Comment by donnaware** [comment]
It sound like you are making good progress.  You don't have to use the BCB code, that was just an example app. You can use the DLL's can make calls to them using Python or whatever code you want. once you have stepped through the process of installing the USB drivers, then the DLL's provide the interface to communicate with the Movidius.  I hope this helps, but the precompiled app is a good way to check that the USB drivers installed OK.

**Comment by donnaware** [comment]
Oh, I just checked and I did not post the precompiled DLL's.  I will do that a little later today.  Then people can use Python or whatever.

**Comment by shuxp** [comment]
yes, that's what I want. Looking forward to see the DLL.
precompiled app worked OK here.
Thanks very much for your help.

**Comment by donnaware** [comment]
Right, that makes sense. I just uploaded the gcc code for the mvnc_api.dll which is basically the windows version of the mvnc_api.a for Linux.  It should work with Python or any program that can load a DLL.

I hope that helps you out.  I actually sort of fiddled with this a while back and then dropped it and moved on so I never really finished it but I posted what I had so I hope it helps someone else get started. It sounds like at least you are on your way.  If the precompiled app worked then your drivers are good.
